sequencer/NetworkPlan.py

```diff
@@ -255,11 +280,15 @@ def network(self):
         return self._network
     
     @property
+    def original_metrics(self):
+        """returns the original (unprocessed) metrics data_frame"""
+        return self._original_metrics
```

Comment:
Oh yeah, now that you mention it there's a lot of property weirdness going on here... May as well clean it up in a different PR.